app/javascript/dashboard/components/widgets/modal/ConfirmDeleteModal.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <modal :show.sync="show" :on-close="closeModal">
+    <woot-modal-header :header-title="title" :header-content="message" />
+    <form @submit.prevent="onConfirm()">
+      <woot-input
+        v-model="value"
+        type="text"
+        :class="{ error: $v.value.$error }"
+        :placeholder="cofirmPlaceHolderText"
+        @blur="$v.value.$touch"
+      />
+      <div class="button-wrapper">
+        <woot-button
+          color-scheme="alert"
+          :is-disabled="!value || !$v.value.isEqual"
```

Comment:
Can't we have something like `$v.error` for the entire form? What I meant is that if and only if form is valid condition can be used here.